Packs/SIGNL4/.pack-ignore

```diff
@@ -0,0 +1,11 @@
+[file:SIGNL4.yml]
+ignore=IN126
+
+[file:README.md]
+ignore=RM106
+
+[file:SIGNL4_image.png]
+ignore=IM111
```

Comment:
Consider using a logo with the required proportions (120x50 pixels) to make it look better.
![image](https://github.com/user-attachments/assets/da02e0a6-1dd3-418c-9e0e-7922101a502c)

Reply:
Thanks and I have added a small logo here.